openfisca_uk/variables/household/attributes.py

```diff
@@ -181,5 +181,41 @@ def formula(household, period, parameters):
             household(
                 "equiv_household_net_income_ahc", period, options=[DIVIDE]
             )
-            < parameters(period).poverty.absolute_poverty_ahc
+            < parameters(period).poverty.absolute_poverty_threshold_ahc
         )
+
+
+class poverty_gap_bhc(Variable):
+    value_type = float
+    entity = Household
+    label = u"Positive financial gap between net household income and the poverty line"
+    definition_period = WEEK
+
+    def formula(household, period, parameters):
+        poverty_line = parameters(
```

Comment:
could an extra formula improve modularity for each `poverty_gap` formula? e.g.:

```
def poverty_gap(threshold, equivalization, net_income):
    poverty_line = threshold / equivalization
    return max_(0, poverty_line - net_income)
```

Reply:
Yeah I like that, though I think it's best to keep all formulas OpenFisca standard - we could add a poverty_line formula for households, and have the poverty gap formula just use that